docs/en/cloud/security/aws-privatelink.md

```diff
@@ -95,199 +130,191 @@ resource "aws_vpc_endpoint" "this" {
-Trying 172.31.25.195...
-# highlight-next-line
-telnet: connect to address 172.31.25.195: No route to host
-Trying 172.31.3.200...
+#### Terraform
+- Change the `aws_vpc_endpoint` resource in Terraform code and set `private_dns_enabled` to `true`:
+```json
+  private_dns_enabled = true
 ```
 
-The error below is likely caused by a missing attached security group for the VPC endpoint that allows ClickHouse ports:
-```response
-telnet iyc9vhhplz.us-east-1.aws.clickhouse.cloud 9440
-Trying 172.31.30.46...
+- Apply the change
 
+## Add Endpoint ID to ClickHouse Cloud organization
 
+### REST API
 
-telnet: connect to address 172.31.30.46: Connection timed out
+Please set environment variables before running curl commands:
+```bash
+PROVIDER=aws
+KEY_ID=<Key ID>
+KEY_SECRET=<Key secret>
+ORG_ID=<please set ClickHouse organization ID>
+ENDPOINT_ID=<Endpoint ID from previous step>
+REGION=<region code, please use AWS format>
 ```
 
-## Shift network traffic to VPC Endpoint
-:::note
-This step switches network traffic from traveliing over the Internet to using the VPC Endpoint.
-:::
+Please set the VPC_ENDPOINT environment variable using data from the previous step.
 
-Before this step:
-```response
-[ec2-user@ip-172-31-29-231 ~]$ nslookup HOSTNAME.clickhouse.cloud
-Server:         172.31.0.2
-Address:        172.31.0.2#53
-
-Non-authoritative answer:
-Name:   HOSTNAME.clickhouse.cloud
-# highlight-next-line
-Address: 44.226.232.172
-Name:   HOSTNAME.clickhouse.cloud
-# highlight-next-line
-Address: 35.82.252.60
-Name:   HOSTNAME.clickhouse.cloud
-# highlight-next-line
-Address: 35.85.205.122
+To add:
+```bash
+cat <<EOF | tee pl_config_org.json
+{
+  "privateEndpoints": {
+    "add": [
+      {
+        "cloudProvider": "aws",
+        "id": "${ENDPOINT_ID}",
+        "description": "An aws private endpoint",
+        "region": "${REGION}"
+      }
+    ]
+  }
+}
+EOF
 ```
 
-```response
-After completion of this step:
-[ec2-user@ip-172-31-29-231 ~]$ nslookup HOSTNAME.clickhouse.cloud
-Server:         172.31.0.2
-Address:        172.31.0.2#53
-
-Non-authoritative answer:
-Name:   HOSTNAME.clickhouse.cloud
-# highlight-next-line
-Address: 172.31.27.78
-Name:   HOSTNAME.clickhouse.cloud
-# highlight-next-line
-Address: 172.31.33.234
-Name:   HOSTNAME.clickhouse.cloud
-# highlight-next-line
-Address: 172.31.8.117
+To remove:
+```bash
+cat <<EOF | tee pl_config_org.json
+{
+  "privateEndpoints": {
+    "remove": [
+      {
+        "cloudProvider": "aws",
+        "id": "${ENDPOINT_ID}",
+        "region": "${REGION}"
+      }
+    ]
+  }
+}
+EOF
 ```
 
-## AWS Console
+Add / remove Private Endpoint to organization
 
-Go to **VPC Endpoints** and right click the VPC Endpoint, then click to **Modify private DNS name**:
+```bash
+curl --silent --user $KEY_ID:$KEY_SECRET -X PATCH -H "Content-Type: application/json" https://api.clickhouse.cloud/v1/organizations/$ORG_ID -d @pl_config_org.json
+```
```

Comment:
I'm not sure what this section is for. Can you leave a description or explanation?